.github/workflows/release.yml

```
@@ -36,3 +36,17 @@ jobs:
           args: release --clean --config .goreleaser.yml
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout copa-action repository
+        uses: actions/checkout@v2
```

nit: please add a tag and comment with the corresponding version

```suggestion
        uses: actions/checkout@12345 # vX.Y.Z
```